docs/CHARTER.md

```diff
@@ -0,0 +1,114 @@
+
+
+# Technical Charter (the “Charter”) for Criticality Score a Series of LF Projects, LLC
+
+**Adopted ___________**
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Criticality Score open source project, which has been established as Criticality Score a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+
+
+## 1. Mission and Scope of the Project
+a. The mission of the Project is to determine a project's criticality score to the influence and importance of a project.
```

Comment:
a. The mission of the Project is to generate criticality scores for open source projects, in order to identify critical projects the open source community depends on so these projects can be proactively secured.